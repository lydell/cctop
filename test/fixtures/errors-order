# Errors should be in line order.
\ cont
	 sub
