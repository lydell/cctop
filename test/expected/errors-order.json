{
  "commands": [],
  "errors": [
    {
      "id": "missing-command-name-line",
      "line": 1,
      "column": 1,
      "command": {
        "name": null,
        "value": "\ncont",
        "lines": [[null, null, 0], [2, 3, 4]],
        "line": null,
        "column": null,
        "endLine": 2,
        "subCommands": []
      }
    },
    {"id": "indent-mixed-space-tab-line", "line": 3, "column": 1}
  ]
}